src/pages/index.js

```diff
@@ -16,20 +17,46 @@ import {
   profileEditForm,
   addCardForm,
   addCardButton,
+  avatarForm,
+  avatarButton,
 } from "../utils/constants.js";
-/**
- * =================================================
- *                INSTANCE CLASS
- * =================================================
- */
-const cardSection = new Section(
-  {
-    items: initialCards, // start with initialCards
-    renderer: (cardData) => cardSection.addItems(createCard(cardData)),
-  }, // use the data to create a card
-  ".cards__list" //refer to the (selector) in section class
-);
-cardSection.renderItems(); // call the renderitems() to show the cards on the page
+
+// API
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "beff3ef6-54fc-4029-9809-d6cc7f6fad90",
+    "Content-Type": "application/json",
+  },
+});
+let cardSection;
+api
+  .getCardList()
+  .then((cards) => {
+    cardSection = new Section(
+      {
+        items: cards, // start with initialCards
+        renderer: (cardData) => cardSection.addItems(createCard(cardData)),
+      }, // use the data to create a card
+      ".cards__list" //refer to the (selector) in section class
+    );
+    cardSection.renderItems();
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+api.getProfileInfo().then((userData) => {
+  userInfo.setUserInfo({
+    title: userData.name,
+    description: userData.about,
+  });
+  userInfo.setAvatarImage({ avatar: userData.avatar });
+});
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**